pypit/arload.py

```diff
@@ -374,7 +374,8 @@ def load_master(name, exten=0, frametype='<None>'):
             if settings.argflag['reduce']['masters']['force']:
                 msgs.error("Master calibration file does not exist:"+msgs.newline()+name)
             else:
-                msgs.error("Could not properly ready Master calibration file:"+msgs.newline()+name)
+                msgs.warn("Could not read Master calibration file:"+msgs.newline()+name)
+                raise IOError
```

Comment:
Instead of raising errors, should we catch them by raising a warning first (as you've done) and then say something like:

`msgs.error("Cannot continue without a master calibration file")`

or something like that?

Reply:
I used this as a jumping off point for #329 .